recipes/pytest-picked/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pytest-picked" %}
+{% set version = "0.2.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "426a5787d0a9398f628302a09aaf71bd1cb2706b38fc78939d18c44279842749" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools
+    - pytest >=3.5.0
+  run:
+    - python >=3.5
+    - setuptools
```

Comment:
Does this project use ``setuptools`` as a run requirement? or just for build?

Reply:
Ah good catch, totally missed that. Thanks!